src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,162 @@
+
+        printLottoWinMessage();
+        decideRanking();
+        printRankingResult();
+
+        computeEarningRate();
+        printEarningRateResult();
+    }
+
+    public void inputPurchaseAmount() {
+        boolean isValidNumber = false;
+
+        do {
+            inputView.printMessage(InputMessage.INPUT_PURCHASE_AMOUNT);
+            inputCost = inputView.inputPurchaseAmount();
+
+            try {
+                parsedCostToInt = lottoService.isValidNumber(inputCost);
+                isValidNumber = true;
+            } catch (IllegalArgumentException e) {
+                outputView.printErrorMessage(e.getMessage());
+            }
+        } while (!isValidNumber);
+    }
+
+    public void printNumberOfPurchaseLotto() {
+        purchasedLottoCount = lottoService.divideInputByLottoPrice(parsedCostToInt);
+        String lottoCountMessage = OutputMessage.PURCHASED_LOTTO_COUNT_MESSAGE
+                .getLottoCountMessage(purchasedLottoCount);
+        outputView.printLottoCountMessage(lottoCountMessage);
+    }
+
+    public void generateLottosByRandomNumber() {
+        this.lottoCollection = lottoService.generateLottoCollection(purchasedLottoCount);
+    }
+
+    public void printAllLottoNumbers() {
+        List<Lotto> lottoNumbers = lottoService.prepareCollectionForPrint(lottoCollection);
+        outputView.printLottoNumbers(lottoNumbers);
+    }
+
+    public void inputWinningNumber() {
+        boolean isValidInput = false;
+
+        do {
+            inputView.printMessage(InputMessage.INPUT_WINNING_NUMBER);
+            winningNumber = inputView.inputWinningNumber();
+            try {
+                validateWinningNumber();
+                isValidInput = true;
+            } catch (IllegalArgumentException e) {
+                outputView.printErrorMessage(e.getMessage());
+            } catch (IllegalStateException e) {
+                outputView.printErrorMessage(e.getMessage());
+            }
+        } while (!isValidInput);
+    }
+
+    public void validateWinningNumber() {
+        // split 전, 검증
+        lottoService.validateInputWinNumber(winningNumber);
+
+        String[] splitWinningNumber = lottoService.splitWinningNumber(winningNumber);
+
+        // split 후, 검증
+        WinningNumber validWinNumber = lottoService.validateSplitWinNumber(splitWinningNumber);
+        this.winningNumberObject = validWinNumber;
+    }
+
+    public void inputBonusNumber() {
+        boolean isValidInput = false;
+
+        do {
+            inputView.printMessage(InputMessage.INPUT_BONUS_NUMBER);
+            bonusNumber = inputView.inputBonusNumber();
+            try {
+                validateBonusNumber();
+                isValidInput = true;
+            } catch (IllegalArgumentException e) {
+                outputView.printErrorMessage(e.getMessage());
+            } catch (IllegalStateException e) {
+                outputView.printErrorMessage(e.getMessage());
+            }
```

Comment:
멀티캐치를 쓰면 가독성이 향상될 것 같아요
```suggestion
            } catch (IllegalArgumentException  | IllegalStateException e ) {
                outputView.printErrorMessage(e.getMessage());
            }
```

Reply:
이렇게 쓸 수도 있었네요 감사합니다!